contributor-guide/modules/ROOT/pages/version-control.adoc

```diff
@@ -1,4 +1,138 @@
-= Version Control (Git)
+= Version Control
 :navtitle: Version Control
 
-*Content under construction.*
+Boost uses the Git version control system for its development. This allows multiple contributors to work on the libraries concurrently, while keeping track of the changes that each person makes. Git is a distributed version control system, which means each contributor has their own local copy of the entire project, including its history.
+
+== Library Repositories
+
+The Boost libraries are modular, and each library has its own separate repository.  This makes it easier to work with the entire collection of Boost libraries.
+
+The official Boost repositories are hosted in GitHub under the https://github.com/boostorg/[] account. Each library has its own repository under this organization. For example, https://github.com/boostorg/json for boost:json[], or https://github.com/boostorg/date_time/ for boost:date_time[].
+
+As with any Git-based project, you can clone the repositories, make changes, commit those changes to your local repository, and push your changes back to the server. If you want to contribute changes to the official Boost libraries, refer to xref:getting-involved.adoc#contribute[Contribute to an Existing Library].
+
+== The Super-project Repository
+
+The Boost https://github.com/boostorg/boost[*Super-project*] repository includes all the libraries as submodules.
+
+In addition to the libraries, you'll find many other repositories under `boostorg` that handle other tasks, for example:
+
+* https://github.com/boostorg/website2022[Website generation]
+* https://github.com/boostorg/cmake[CMake support]
+* https://github.com/boostorg/regression[Regression testing]
+* https://github.com/boostorg/release-tools[Release tools]
+
+CMake is one of the most popular of the https://github.com/boostorg/boost/tree/master/tools[boost/tools] directory. You will find other tools used for tasks such as building and testing Boost libraries. 
+
+The Super-project has both *master* and *develop* branches, which operate somewhat differently than for individual libraries. 
+
+New features are added and bugs are fixed in library *develop* branches. When the *develop* branch is complete, it is merged with the library *master* branch. 
```

Comment:
A branch doesn't get "complete" - code changes in develop are tested and once the code is considered to be stable (e.g. CIs have completed, other projects are building without errors, or whatever your criteria is), then it's merged to master.

Reply:
ok, reworded